repo2docker/buildpacks/docker.py

```diff
@@ -15,10 +16,21 @@ def detect(self):
         return os.path.exists(self.binder_path("Dockerfile"))
 
     def render(self, build_args=None):
-        """Render the Dockerfile using by reading it from the source repo"""
+        """
+        Render the Dockerfile used by reading it from the source repo
+
+        If an appendix is configured, it will be appended to the Dockerfile
+        found in the source repo.
+        """
         Dockerfile = self.binder_path("Dockerfile")
```

Comment:
Trying to understand if there was an issue with this code, I reacted to seeing the reference `self.dockerfile` be removed and a hardcoded mention of `"Dockerfile"` remain. In practice, `self.dockerfile` is hardcoded to "Dockerfile"` and only some references to that string relied on the placeholder name.

If this refactoring had been made, or `"Dockerfile"` would been consistently used as a hardcoded string, I wouldn't been confused.

```suggestion
        Dockerfile = self.binder_path(self.dockerfile)
```

Reply:
@consideRatio i didn't even notice I replaced that! I just removed it, no reason to keep that.